package.json

```diff
@@ -8,7 +8,8 @@
     "build": "sh -ac '. ./.env.${REACT_APP_ENV}; react-scripts build'",
     "build:test": "REACT_APP_ENV=test yarn build",
     "build:production": "REACT_APP_ENV=production yarn build",
-    "test": "firebase emulators:exec --project aiscope-labelling-app-test --import mockData 'react-scripts test --watchAll=false --detectOpenHandles --forceExit'",
+    "test": "react-scripts test --watchAll=false --detectOpenHandles --forceExit",
+    "test:emulator": "firebase emulators:exec --project aiscope-labelling-app-test --import mockData 'react-scripts test --watchAll=false --detectOpenHandles --forceExit'",
```

Comment:
These tests are not going to be executed by the pipeline, is this expected?

Reply:
Yes, it is expected.

The tests with firebase emulator are considerable expensive to run (in terms of time) and once the `DatabaseService` has been removed there is no need to run the tests using the emulator.

To keep the possibility and the approach to run the tests with the emulator I decided to keep it as a "secondary" run script (`test:emulator`).